kodilla-hibernate/src/main/java/com/kodilla/hibernate/invoice/Product.java

```diff
@@ -0,0 +1,54 @@
+package com.kodilla.hibernate.invoice;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotNull;
+import java.util.ArrayList;
+import java.util.List;
+
+@Entity
+@Table(name = "PRODUCT")
+public class Product {
+    private String name;
+    private int id;
+    private List<Item> items = new ArrayList();
```

Comment:
why product has a reference to items? It does not make sense to me. Item should have reference to product, but not in the opposite direction.